tools/alt-testing/create-projects.sh

```diff
@@ -0,0 +1,43 @@
+set -e
+
+# Expecting
+# TF_PROJECT_ID=
+# KRM_PROJECT_ID=
+# DM_PROJECT_ID=
+
+BILLING_ACCOUNT=[BILLING_ACCOUNT]
+FOLDER_ID=[FOLDER_ID]
+CLUSTER_ID=[CLUSTER_ID]
+ZONE=[ZONE]
```

Comment:
looks like all of these variables are required, is the intention that the ones that are commented out are set as env vars while the remaining must be set in the script? can we be more consistent?

Reply:
I see that the project variables are referenced in `delete-project.sh` as well, it makes sense for those to be global in that case. perhaps just add a comment confirming that it's intentional that some variables must be defined in the script while others should be global

Reply:
I added the comment in the file to clarify - this script can be included as source and the variables above are coming from the parent `test_alternatives` script, while the 3 other variables should be defined here.